packages/datadog-plugin-http2/src/server.js

```diff
@@ -3,11 +3,52 @@
 // Plugin temporarily disabled. See https://github.com/DataDog/dd-trace-js/issues/312
 
 const Plugin = require('../../dd-trace/src/plugins/plugin')
+const { storage } = require('../../datadog-core')
+const web = require('../../dd-trace/src/plugins/util/web')
+const { incomingHttpRequestStart } = require('../../dd-trace/src/appsec/gateway/channels')
 
 class Http2ServerPlugin extends Plugin {
   static get name () {
     return 'http2'
   }
+
+  constructor (...args) {
```

Comment:
The class could instead extend from `HttpServerPlugin` and use `apm:${this.constructor.name}` as the prefix for events.

Reply:
We can do that in a future PR.